packages/web-app/src/components/appli/Entry/Properties.jsx

```diff
@@ -188,18 +228,15 @@ const Properties = () => {
           />
         )}
       </Box>
-
-      <Property
-        label={formatMessage({ id: 'Average rating' })}
-        icon={<EmptyStarIcon fontSize="large" color="primary" />}
-      />
-      <SmallRatingsWrapper>
-        <Ratings
-          access={access}
-          interest={interest}
-          progression={progression}
-        />
-      </SmallRatingsWrapper>
+      {!!access && !!interest && !!progression && (
```

Comment:
`!!access || !!interest || !!progression && (` instead no ?

The Ratings component already check if one of the three props is missing : it should be displayed if at least one of these is available I think.

Reply:
Modified